pkgs/tools/networking/mosh/default.nix

```diff
@@ -11,11 +11,13 @@ stdenv.mkDerivation rec {
     sha256 = "05hjhlp6lk8yjcy59zywpf0r6s0h0b9zxq0lw66dh9x8vxrhaq6s";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkg-config makeWrapper ];
+  nativeBuildInputs = [ autoreconfHook pkg-config makeWrapper protobuf ]
+    ++ (with perlPackages; [ perl IOTty ]);
   buildInputs = [ protobuf ncurses zlib openssl bash-completion ]
```

Comment:
I guess protobuf is still needed in buildInput, but did you check by chance?

Reply:
Just checked, and it is needed. The binaries dynamically link to libprotobuf.

Reply:
Yeah, that's what I thought. Thanks for checking 🙂

Reply:
i use diffoscope to compare the result when turning strictDeps on